gems/pending/middleware/events/middleware_event_monitor.rb

```diff
@@ -0,0 +1,22 @@
+class MiddlewareEventMonitor
+  def initialize(ems)
+    @ems           = ems
+    @alerts_client = ems.alerts_connect
+  end
+
+  # Each poll is performed from the time of the most recently caught event or 1 minute back for the first poll.
+  # This gives us some slack if hawkular events are timestamped behind the miq server time.
+  # TODO: Is there a danger here?  This assumes all Hawkular events at time T are fetched in one polling call.
+  # TODO: Can we store start_time and resume where we left off?  We miss events when we're not running...
+  def poll
```

Comment:
maybe I'm not getting it, but this method does not poll?! 
It just fetches one batch of events. If it would poll, it would loop.